### ISSUE TITLE
confusion about the performance on IEMOCAP

### ISSUE THREAD

**Issue by 201528014227051** [gkoumasd/MSAF#4]
Thank u for the great work.
When I run the MARN model on the IEMOCAP data set, I find that the performance of in the output .csv is higher than the results reported in the paper.
the accuracy is 
0.824627
F1 score for every class is 0.7105674147704142, 0.8270833752719677, 0.8452668586306232, 0.8754207538192843
am I missing something?

**Comment by gkoumasd** [comment]
Hi,

thank you for the feedback.
Indeed, the performance is too high for a multi-class classification task.
According to the literature, MULT is the sota approach. But, still, your results are pretty higher from Mult's performance.
Have you changed the configuration settings (i.e., label) from sentiment to emotion?

**Comment by 201528014227051** [comment]
I have changed the label.
After comparing the hyperparameter settings, I changed the attn_num to 1, It seems overfitting if atten_num is larger. and another difference is  the epoch number in run.ini file, I changed it from 1 to 5
